### ISSUE TITLE
Incorrect author for citations

### ISSUE THREAD

**Issue by bryceerobertson** [StampyAI/stampy-ui#696]
e.g. plex is listed as the author of this video (https://www.youtube.com/watch?v=zc4gKH6xTBw), but he most definitely isn't.

![Image](https://github.com/StampyAI/stampy-ui/assets/98026499/51438a3c-b422-4bc4-a605-3ac38c0a84da)

**Comment by zarSou9** [comment]
@mruwnik do you know why this might be?

**Comment by mruwnik** [comment]
yes. It's because it's from his YT channel. It might be worth not showing the author for YT videos. Or fixing the scraper would be even better

**Comment by bryceerobertson** [comment]
@melissasamworth I've seen other incorrect citations, like [this one](https://arbital.com/p/general_intelligence/). think this should be P0.

**Comment by Ryan-Knowles** [comment]
@bryceerobertson going to look into this and take a shot at a fix

**Comment by mruwnik** [comment]
This is caused by how the ARD looks for YT authors [here](https://github.com/StampyAI/alignment-research-dataset/blob/main/align_data/sources/youtube/youtube.py#L91). These vidoes are from plex's playlist, so he gets credited as the author

**Comment by mruwnik** [comment]
Myles has a hacky short term solution [here](https://github.com/StampyAI/stampy-ui/pull/734/files), which is to just not show YT authors

**Comment by bryceerobertson** [comment]
The problem is I've seen it also happen for non-YouTube sources like https://arbital.com/p/general_intelligence/

**Comment by mruwnik** [comment]
yup. That's correct 


![Image](https://github.com/StampyAI/stampy-ui/assets/3942390/b0926fbb-dd5f-44ca-9964-7d98353bceab)

https://huggingface.co/datasets/StampyAI/alignment-research-dataset/raw/main/arbital.jsonl

**Comment by bryceerobertson** [comment]
Ah right, I thought it must be wrong haha. Ok good, so it's only YouTube then :)
